main/README.md

```diff
@@ -60,6 +94,8 @@ If the chosen `db_password_strategy` is `aws-secrets-manager`, this terraform mo
 | sg | ID of Beekeeper SG. |
```

Comment:
Now that we won't need to configure anything in the app, theses outputs can be removed right?

Reply:
Removed all but `output.sg` which looks like it is being used still by the implementation of `beekeeper-slack-lambda`. Depending on how we refactor the `slack-lambda` that could be removed as well.